gobblin-runtime/src/main/java/gobblin/scheduler/JobScheduler.java

```diff
@@ -144,13 +147,25 @@ public JobScheduler(Properties properties, SchedulerService scheduler)
     this.waitForJobCompletion = Boolean.parseBoolean(
         this.properties.getProperty(ConfigurationKeys.SCHEDULER_WAIT_FOR_JOB_COMPLETION_KEY,
             ConfigurationKeys.DEFAULT_SCHEDULER_WAIT_FOR_JOB_COMPLETION));
+
+    if (this.properties.containsKey(ConfigurationKeys.JOB_CONFIG_FILE_GENERAL_PATH_KEY)) {
```

Comment:
We should update our Getting-started/Scheduler documents about this cool feature!